kafkaesk/ext/logging/handler.py

```diff
@@ -0,0 +1,111 @@
+from .formatter import PydanticLogModel
+from .record import PydanticLogRecord
+from pydantic import BaseModel
+from typing import IO
+from typing import Optional
+from typing import Tuple
+
+import asyncio
+import kafkaesk
+import logging
+import sys
+
+
+class InvalidLogFormat(Exception):
+    ...
+
+
+class PydanticStreamHandler(logging.StreamHandler):
+    def __init__(self, stream: Optional[IO[str]] = None):
+        super().__init__(stream=stream)
+
+    def format(self, record: logging.LogRecord) -> str:
+        message = super().format(record)
+
+        if isinstance(message, BaseModel):
+            message = message.json()
+
+        return message
+
+
+class LogQueue:
+    def __init__(self, app: kafkaesk.Application):
+        self._queue: asyncio.Queue[Tuple[str, BaseModel]] = asyncio.Queue()
```

Comment:
Would it make sense to bound the max length of the queue? Otherwise, if there is an outage or whatever with Kafka with the downside effect of having large timeouts when data is published this queue could fill up with data quickly.

Reply:
That's a good idea.  Any thoughts on what that max length should be?  What actions should we take if the queue fills up?  Right now the handler will write a message to stderr.